### PR TITLE
Branch ug examples

### LINKED ISSUE #135: UG: Forgot to update what the example means
image

### PR DESCRIPTION
**Pull request by Respirayson**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -336,19 +358,37 @@ You can edit an existing student's fields in F.A.K.E.J.A.R.V.I.S.
...
-[//]: # (* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed 
-person list. The index **must be a positive integer** 1, 2, 3, …​   THIS CAN BE IN THE GLOSSARY)
+<div class="alert alert-info"> :information_source: Note: 
+
+* Edits the person at the specified `INDEX`. 
+* The index refers to the index number shown in the displayed person list. 
+* The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the optional fields must be provided. (i.e NAME, PHONE, EMAIL, TELEGRAM_HANDLE)
+* Editing tags overwrites existing tags; it's not cumulative. To remove all tags, use `t/` without specifying any tags after it.
+* You can remove all the person’s tags by typing `t/` without specifying any tags after it.
```
Comment (Originalidk) [inline docs/UserGuide.md]: Perhaps remove all tags should not be repeated? I think it should be in a line of its own.

**Review by Originalidk — COMMENTED** [review]
Just a small nit. Other than that LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -310,14 +310,26 @@ You can add a student to F.A.K.E.J.A.R.V.I.S.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL th/TELEGRAM_HANDLE [t/TAG]…​`
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@u.nus.edu th/johnny01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@u.nus.edu th/itsybetsyspider p/1234567 t/criminal`
+<div markdown="span" class="alert alert-info"> :information_source: Note: You can add any number of tags to a person! (including 0) </div>
 
+> Example 1:
```
Comment (raydenlim) [inline docs/UserGuide.md:315]: u forgot the `gt/` part. Its after the `t/`

Comment (raydenlim) [reply docs/UserGuide.md:315]: unless its meant to not be initialised

**Review by raydenlim — APPROVED** [review]
minor details aside, LGTM

**Pull request merged**